docs/Users/Install.rst

```diff
@@ -166,6 +166,28 @@ https://docs.python.org/3.3/install/#alternate-installation.
     http://stackoverflow.com/questions/14149422/using-pip-behind-a-proxy
     for further clarification.
 
+Error handling
+~~~~~~~~~~~~~~~~
+
+In case you are getting
+``ValueError:('Expected version spec in', 'appdirs ~=1.4.0', 'at', ' ~=1.4.0')``
+then don't panic. It happens when you are using an outdated version of PIP
```

Comment:
I am not sure. I thought here we are addressing pip as a module name rather than a command :P What do you think?